// Exercice 2: GÃ©rer les erreurs avec try...catch


try {
    URL("http//gomle.com");
} catch (error) {
    console.log("Il y a une erreur ");
}